MAINTAINERS.md

```diff
@@ -0,0 +1,51 @@
+# Helm Maintainers
+
+This document explains the leadership structure of the Kubernetes Helm project, and list the current project maintainers.
```

Comment:
That list currently has members that have expressed that they don't want to be "core maintainers". I would be okay with an external list as long as we can make that definitive. Thoughts?